Doc/library/unittest.mock.rst

```diff
@@ -514,6 +514,18 @@ the *new_callable* argument to :func:`patch`.
             >>> mock.call_count
             2
 
+        For :class:`AsyncMock` the :attr:`call_count` is only iterated if the function
+        has been awaited:
+
+            >>> mock = AsyncMock()
+            >>> mock()
```

Comment:
This would cause a warning that the mock was not awaited and there is a failure in CI due to the repr output. Perhaps below would help where we skipped doctest in another similar example.

```python
>>> mock()  # doctest: +SKIP
<coroutine object AsyncMockMixin._mock_call at ...>
```